source/_docs/https-load-balancer.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Supported Methods for Configuring an HTTPS Load Balancer
+description: Explore different solutions for enabling HTTPS on all environment URLs (including Multidev) associated with your organization.
+categories: [developing]
+tags: [domains, organizations]
+---
+
+## Shared Load Balancer and CloudFlare's Universal SSL (Free)  
```

Comment:
CloudFlare is a CDN, not a shared load balancer. We should probably not put this first, since most of the people who encounter this will not be using CloudFlare as a service. However, we can mention it as an example of how to use a CDN in this context, and link to our guide.